components/terrain/terraindrawable.cpp

```diff
@@ -4,13 +4,19 @@
 
 #include <components/sceneutil/lightmanager.hpp>
 
+#include "compositemaprenderer.hpp"
+
 namespace Terrain
 {
 
 TerrainDrawable::TerrainDrawable()
 {
 }
 
+TerrainDrawable::~TerrainDrawable()
+{
+}
```

Comment:
I think the constructor and destructor can be declared as
```cpp
TerrainDrawable() = default;
~TerrainDrawable() = default;
```
in the header file and these empty declarations aren't necessary.